86c5cuvtx/feature/observability

AmitMizrachi618 opened the pull request:
Changes to UUID Utils to work with dynamic bytes length, and add/extract suffixes.

frankos333 committed:
Adding add and extract suffix methods and making int to bytes flexible to uuid length.

(#86c5cuvtx)

frankos333 committed:
Removing useless type hints that we do not need.

(#86c5cuvtx)

frankos333 committed:
Bugfix where length didnt have a default value.

(#86c5cuvtx)

frankos333 committed:
Moving http utils from utils internal to utils external, his rightful place.
Moving http utils from utils internal to utils external, his rightful place.

(#86c5cuvtx)

AmitMizrachi618 committed:
Updated the value error message raised by extract suffix.
Updated the value error message raised by extract suffix.

(#86c5cuvtx)

AmitMizrachi618 committed:
Removing HttpUtils as its no longer in use here in this library.
Removing HttpUtils as its no longer in use here in this library.

(#86c5cuvtx)

AmitMizrachi618 committed:
Adding json property seracher class which will do bfs for a set of values in a dict and return a dict with the found ones.
Adding json property seracher class which will do bfs for a set of values in a dict and return a dict with the found ones.

(#86c5cuvtx)

yonatankarimish commented on norman_utils_external/json_property_searcher.py:
```diff
@@ -0,0 +1,27 @@
+from collections import deque
+
+
+class JsonPropertySearcher:
```
Design: write yourself according to Norman standards, and not the basic gpt solution

yonatankarimish commented on norman_utils_external/uuid_utils.py:
Design: see matching comment on norman_utils_internal observability_middleware. You probably do not need these additions.

yonatankarimish reviewed (changes requested):
See inline comments

AmitMizrachi618 committed:
Made the BFS written the Norman Way. Lets Norman it!.
Made the BFS written the Norman Way. Lets Norman it!.

(#86c5cuvtx)

AmitMizrachi618 committed:
Reverting UUID utils as we no longer need pairing ids.
Reverting UUID utils as we no longer need pairing ids.

(#86c5cuvtx)

AmitMizrachi618 committed:
Adding union of set lisst and tuple, adding set to isinstance, extra line of space.
Adding union of set lisst and tuple, adding set to isinstance, extra line of space.

(#86c5cuvtx)

yonatankarimish reviewed (approved):
Approved

yonatankarimish reviewed (dismissed):
Approved

yonatankarimish committed:
Moved name utils from norman authenticate to norman utils external.

(#86c6wa00a)

yonatankarimish committed:
Merge pull request #11 from norman-ml/86c6wa00a/tech-debt/sdk-tests

Initial groundwork for sdk tests.

(#86c6wa00a)
